src/devices/video/epic12.h

```diff
@@ -211,34 +212,34 @@ class epic12_device : public device_t, public device_video_interface
 		}
 
 
-		uint8_t b, g, r, t;
+		u8 b, g, r, t;
 	};
 
 	union colour_t
 	{
 		clr_t trgb;
-		uint32_t u32;
+		u32 data;
```

Comment:
This is seriously not endian-safe, could we have that fixed?

Reply:
Now fixed, and have a no issue currently.

Reply:
I seem to remember trying to use it as a union almost halved the bench speed at the time anyway